Library/Formula/mandoc.rb

```diff
@@ -0,0 +1,74 @@
+class Mandoc < Formula
+  homepage "http://mdocml.bsd.lv"
+  url "http://mdocml.bsd.lv/snapshots/mdocml-1.13.3.tar.gz"
+  sha256 "23ccab4800d50bf4c327979af5d4aa1a6a2dc490789cb67c4c3ac1bd40b8cad8"
+
+  head "anoncvs@mdocml.bsd.lv:/cvs", :module => "mdocml", :using => :cvs
+
+  depends_on "sqlite"
+
+  option "without-db", "Only install the mandoc/demandoc utilities."
+  option "without-cgi", "Don't build man.cgi (and extra CSS files)."
```

Comment:
For the database option, it eliminates the sqlite dep (which I need to add to the `depends_on`), and avoid the `makewhatis`/`mandoc.db` issue discussed above, and just installs the `mandoc` utility (which is basically a `groff` replacement), and `demandoc`, a deroffer.

As, for `man.cgi`, I honestly haven't a clue how it works (or know anyone who uses it, or the old version), and I almost left it out by default. But with it, it also installs a couple extra CSS templates which are useful for formatting static HTML manages, so I left it in.